### ISSUE TITLE
[iOS] Notification sound does not stop on cancelNotification()/cancelDisplayedNotification()

### ISSUE THREAD

**Issue by marcjulianfleck** [notifee/react-native-notifee#384]
If I trigger a notification on iOS from the setBackgroundMessageHandler() of Firebase with a custom sound - which is around 25 seconds - and after some time call 
`notifee.cancelNotification("OWN_ID") `
or 
`notifee.cancelDisplayedNotification("OWN_ID")`
the notification gets removed. However, the sound keeps playing till the end. On the other hand, if I press on the notification, the sound stops immediately. 

This is how I display/start the notification:
```
notifee.displayNotification({
    id: "OWN_ID",
    title: "SOME_TITLE",
    body: ""SOME_BODY",
    ios: {
      sound: "customSound.mp3"
    }
});
```

Anybody solved this issue yet?

**Comment by mikehardy** [comment]
A quick search on this one indicates it may have different behavior depending on whether it is the simulator or a real device, and varying on iOS versions

@marcjulianfleck - can you specify the exact testing environment (versions, real device or simulator) when you see this behavior?

https://stackoverflow.com/questions/19540516/how-stop-local-notification-sound-after-cancel-local-notification
https://stackoverflow.com/questions/4762816/is-it-possible-to-stop-the-sound-of-a-uilocalnotification-from-playing-when-the/27238910#27238910

**Comment by marcjulianfleck** [comment]
@mikehardy yes, I already saw a few similar issues on stackoverflow. However, it seems not be fixed after iOS 8.0 in my case.

My environment:

- iPhone X (real device) with iOS 14.7.1
- Notifee Version: 2.0.0
- react-native Version: 0.64.2
- react-native-firebase Version: 12.7.3 (In case it matters since I show the notification out of the BackgroundMessageHandler)

**Comment by helenaford** [comment]
Mmm interesting, I think this is out of our control 😕
